flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/history/HistoryServerArchiveFetcher.java

```diff
@@ -37,6 +37,7 @@
 import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.databind.JsonNode;
 import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.databind.ObjectMapper;
 
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
I think you should beware of using third party (non-shaded) deps here. Can this be implemented without apache commons?

Reply:
Thank you, I repleace it by own class `JobArchiveEvent`.